docs/guides/admin/docs/configuration/firewall.md

```diff
@@ -0,0 +1,83 @@
+Firewall
+========
+
+Opencast relies on a lot of network communication and although not strictly necessary,
+it usually makes sense to configure a firewall for additional protection
+
+This describes a lot of the communication happening between servers in the most common use cases.
+In certain set-ups and with certain types of integrations, these may deviate slightly.
+
+General rules are:
+
+- Users communicate with Opencast via HTTP(S)
+- Capture agents communicate with Opencast via HTTP(S). All communication is established by the agent.
+- Opencast nodes communicate among each other via HTTP(S)
+- Often Elasticsaecrh and ActiveMQ are run on the admin node since most communication then happens on the server
+    - The admin node communicates with Elasticsaecrh exclusively
+    - Some messages are sent to ActiveMQ from other nodes
+- All servers should get storage access
```

Comment:
`network storage access` perhaps?  Something to make it clear that something like NFS is required.